### ISSUE TITLE
`pipe` CLI shall allow to list the users/groups/objects permissions globally

### ISSUE THREAD

**Issue by sidoruka** [epam/cloud-pipeline#911]
**Background**
The main idea is to automate the retrieval of:
* Which objects are accessible by a user?
* Which objects are accessible by a group?
* Which user/group have access to the object?

**Approach**

**Other options**

**Comment by mzueva** [comment]
@SilinPavel Could you please assist with implementation of new API methods:
- load all permissions for user/group (including inherited permissions)

**Comment by ekazachkova** [comment]
**Approach**

1. Which objects are accessible by a user?
To solve this problem we could use API method `POST /entities` introduced in this PR #949 and introduce a new CLI command `pipe view-user-objects`.

Usage example:
```
$ pipe view-user-objects <USER_NAME>
+--------------+------------+
| Type         | Name       | 
+--------------+------------+
| DATA_STORAGE | storage_1  | 
+--------------+------------+
| PIPELINE     | pipeline_1 | 
+--------------+------------+
```
2. Which objects are accessible by a group?
To solve this problem we could use API method `POST /entities` introduced in this PR #949 and introduce a new CLI command `pipe view-group-objects`.

Usage example:
```
$ pipe view-group-objects <GROUP_NAME>
+--------------+------------+
| Type         | Name       | 
+--------------+------------+
| DATA_STORAGE | storage_1  | 
+--------------+------------+
| PIPELINE     | pipeline_1 | 
+--------------+------------+
```
3. Which user/group have access to the object?
In this case we could use existing CLI command `pipe view-acl`.

Usage example:
```
$ pipe view-acl pipeline_1 -t pipeline
+-----------+-----------+----------------------+------+
|       SID | Principal |                Allow | Deny |
+-----------+-----------+----------------------+------+
| ROLE_USER |     False | Read, Write, Execute |      |
+-----------+-----------+----------------------+------+
```
But the implementation is too old and does not respect permissions for object parents. To fix it we could use `GET /permissions` instead of `GET /grant`.

**Comment by NShaforostov** [comment]
@ekazachkova I've tried to verify this feature. In general, it works well.
Some notes about it:
1. No tools are printed in the full object accessible list for any user (including admins). And `pipe view-user-objects -t tool <user_name>` prints empty table.
2. No Docker registries are printed in the full object accessible list for any user (including admins). And `pipe view-user-objects -t docker_registry <user_name>` prints empty table.
3. The same situation as in items 1, 2 is with the `pipe view-group-objects` and options `-t tool`/`-t docker_registry`
4. Could we "improve" the error message for the `pipe view-group-objects` command? Currently, in both cases - if the specified group doesn't exist (incorrectly specified name, e.g.) or there are no accessible objects for the specified group - the same low-informative error message outputs: `Error: Failed to load entities available for '<GROUP_NAME>'`

**Comment by mzueva** [comment]
@NShaforostov regarding item 4: some groups may not be registered in DB (e.g. loaded from SAML) so I'd suggest not throw error, but just log `No permissions available for group <>` for both cases.

**Comment by NShaforostov** [comment]
@mzueva, I agree. But maybe to change the word `permissions` to `accessible objects` in the suggested message?

**Comment by maryvictol** [comment]
I've tried to verify new commands through the pipe CLI (OS Windows). 
Command pipe view-user-objects <USER_NAME> (pipe view-group-objects <GROUP_NAME>) gives error:
![image](https://user-images.githubusercontent.com/47791878/82033854-03c1ee00-96a6-11ea-83ed-7645674082a6.png)

**Comment by sidoruka** [comment]
@mzueva @ekazachkova please take a look into the @maryvictol comments asap.

**Comment by NShaforostov** [comment]
Docs were updated via #1184
